tests/acceptance/new-test.js

```diff
@@ -363,4 +363,76 @@ describe('Acceptance: ember new', function() {
     expect(file('app/templates/application.hbs'))
       .to.contain("Welcome to Ember");
   }));
+
+  describe('verify fictures', function() {
+    it('app + npm', co.wrap(function *() {
+      yield ember([
+        'new',
+        'foo',
+        '--skip-npm',
+        '--skip-bower',
+        '--skip-git',
+      ]);
+
+      [
+        '.travis.yml',
+      ].forEach(filePath => {
```

Comment:
Not in this PR, but the next one that has README checks will make use of the loop. Future planning.